src/java/org/apache/cassandra/schema/SchemaPushVerbHandler.java

```diff
@@ -35,6 +35,7 @@
  * (which is going to act as coordinator) and that node sends (pushes) it's updated schema state
  * (in form of mutations) to all the alive nodes in the cluster.
  */
+@Deprecated
```

Comment:
This will explode on rebase because of [CASSANDRA-18912](https://issues.apache.org/jira/browse/CASSANDRA-18912) We need to add `since` and maybe even a bit more info like why it was deprecated and when it is for removal (if ever)